### PR TITLE
Add Pep to the odh-admin group

### PR DESCRIPTION
**Pull request by codificat**
This is to add myself to the `odh-admin` group, with the goal of getting access to the osc cluster to use it for CD for the ODH CNBi functionality.

Also taking the opportunity to sort the user list in that group alphabetically.

### CODE REVIEW

**Comment by codificat** [comment]
/cc @tumido 
as discussed in the services call earlier today

File: cluster-scope/base/user.openshift.io/groups/odh-admin/group.yaml [hunk 1]
```diff
@@ -3,17 +3,18 @@ kind: Group
...
-  - crobby
+  - maulikjs
   - nakfour
   - rimolive
-  - humairak
-  - HumairAK
-  - 4n4nd
   - tumido
-  - hemajv
+  - vpavlin
```
Comment (HumairAK) [inline cluster-scope/base/user.openshift.io/groups/odh-admin/group.yaml:20]: @codificat can I ninja your edit and request you remove the following: 

- vpavlin
- maulikjs
- nakfour
- 4n4nd

Comment (HumairAK) [reply cluster-scope/base/user.openshift.io/groups/odh-admin/group.yaml:20]: if you don't mind - feel free to add as a separate commit

Comment (codificat) [reply cluster-scope/base/user.openshift.io/groups/odh-admin/group.yaml]: Sure! done (new commit added)

**Pull request merged**